apps/chat/src/components/Chat/TalkTo/TalkToCard.tsx

```diff
@@ -358,8 +371,22 @@ export const TalkToCard = ({
             <ReplayAsIsIcon size={iconSize} />
           )}
           {!isPseudoModel(entity.reference) &&
-            entity.reference !== REPLAY_AS_IS_MODEL && (
-              <ModelIcon entityId={entity.id} entity={entity} size={iconSize} />
+            entity.reference !== REPLAY_AS_IS_MODEL &&
+            isApplicationsSharingEnabled && (
```

Comment:
we fon't need this condition
```suggestion
            (
```